index.html

```diff
@@ -68,16 +68,16 @@ <h3>Backgrounds</h3>
       </section>
       <form class="save-form">
         <input class= "user-input user-input-js" type="text" placeholder="Name This Outfit" value="">
-        <button class="button save-button" id="save-button" type="button" name="save">Save</button>
+        <button class="button save-button save-button-js" id="save-button" type="button" name="save">Save</button>
       </form>
     </div>
     <div class="column">
       <h3>Saved Outfits</h3>
       <section id="saved-outfits" class="saved-outfits">
-        <div class="outfit-card">
```

Comment:
the html below will be removed now that we have the element info in the JS insertadjacentHTML function.